samples/subsys/usb/cdc_acm/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _usb_cdc_acm:
+
+USB CDC ACM Sample Application
+####################################
+
```

Comment:
Does this sample app only work on specific boards?  If so, you should list them here.